docs/source/conf.py

```diff
@@ -19,14 +19,14 @@
 
 # -- Project information -----------------------------------------------------
```

Comment:
When we update the docs, we'll need to remember to update this information. I think for simplicity let's integrate these changes now, then come back later.

Reply:
Sounds good, I think if we do this merge on my branch, then I can open up some smaller pull requests for various bug fixes that we find along the way 🐞 🧰  (bug fix 😛 ) and for the docs changes.